metrics/influx/influx.go

```diff
@@ -0,0 +1,198 @@
+package metrics
+
+import (
+	"os"
+	"time"
+
+	"github.com/golang/glog"
+	coreInflux "github.com/influxdata/influxdb/client/v2"
+	coreMetrics "github.com/prebid/prebid-server/metrics"
+
+	"context"
+	"github.com/rcrowley/go-metrics"
+	"strconv"
+)
+
+// These are the names of Measurements which we export to Influx.
+const (
+	AUCTION_REQUEST_COUNT    = "prebidserver.auction_request_count"
+	AUCTION_RESPONSE_COUNT   = "prebidserver.auction_response_count"
+	AUCTION_REQUEST_DURATION = "prebidserver.auction_request_duration_nanos"
+
+	BIDDER_REQUEST_COUNT    = "prebidserver.bidder_request_count"
+	BIDDER_RESPONSE_COUNT   = "prebidserver.bidder_response_count"
+	BIDDER_REQUEST_DURATION = "prebidserver.bidder_request_duration_nanos"
+	BID_PRICES              = "prebidserver.bid_response_cpm_cents"
```

Comment:
Not seeing any data recorded against `prebidserver.bid_response_cpm_cents` or `prebidserver.bid_count` measurements.